beater/server.go

```diff
@@ -55,7 +41,7 @@ func (s *Server) create(successCallback func([]beat.Event), host string) *http.S
 
 	// TODO: Remove or find nicer way before going GA
 	if *debugHandler {
-		logp.Warn("Debug handler enabled")
+		logp.Warn("Debug handler Enabled")
```

Comment:
I think it should not matter, as this part will be removed anyways in a follow up PR.